modular_skyrat/modules/customization/modules/clothing/under/misc.dm

```diff
@@ -686,6 +686,29 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
 	icon_state = "christmasfemaleg"
 
+/obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/demon
+	name = "demonic suit"
+	desc = "Stylish enough to impress the devil."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
+	icon_state = "lucifer"
+	obj_flags = UNIQUE_RENAME
+	unique_reskin = list("Lucifer" = "lucifer",
+						"Justice" = "justice",
+						"Malina" = "malina",
+						"Pandemonica" = "pande",
+						"Modeus" = "modeus",
+						"Zdara" = "zdara",
+						"Cereberus" = "cereberus",
+						)
+
+/obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/demon/beeze
+	name = "designer demonic suit"
```

Comment:
I'm alright with everything but the fact that it literally says "demon" and "demonic" in it. Make it say something like "zolmachi" instead, but the word "demon" was ruled out, I even got Xyel's opinion on it and it was pretty categoric.

Reply:
Zolmachi is a language, why would an article of clothing that's not race-specific or whatever, be named after a language? As I stated before, Zolmachi was a language that was integral to part of a character, so that being called "demon" was something to look out for. This is a piece of clothing, it's very easily categorically different things. It's literally just called demonic for reference sake, it's not that big of a deal, I see no reason why this matters in the slightest.

Reply:
Also, I should point out there are other things called "demon" in-game, and people aren't freaking out. This doesn't matter, at all.